src/other_menus.c

```diff
@@ -23,36 +23,402 @@ struct unkStruct_203B35C
 };
 extern struct unkStruct_203B35C *gUnknown_203B35C;
 
-extern struct UnkTextStruct2 gUnknown_80E653C;
-
-extern struct UnkTextStruct2 gUnknown_80E66BC;
-extern struct MenuItem gUnknown_80E66D4[];
-extern struct UnkTextStruct2 gUnknown_80E6748;
-extern struct MenuItem gUnknown_80E6760[];
-extern struct UnkTextStruct2 gUnknown_80E67D4;
-extern struct MenuItem gUnknown_80E67EC[];
-extern struct UnkTextStruct2 gUnknown_80E689C;
-extern struct MenuItem gUnknown_80E68B4[];
-extern struct UnkTextStruct2 gUnknown_80E6938;
-extern struct MenuItem gUnknown_80E6950[];
-extern struct UnkTextStruct2 gUnknown_80E69B0;
-extern struct MenuItem gUnknown_80E69C8[];
-extern struct UnkTextStruct2 gUnknown_80E6A10;
-extern struct MenuItem gUnknown_80E6A28[];
-extern struct UnkTextStruct2 gUnknown_80E6A74;
-extern struct MenuItem gUnknown_80E6A8C[];
-extern struct UnkTextStruct2 gUnknown_80E6B78;
-extern struct MenuItem gUnknown_80E6B90[];
-extern struct UnkTextStruct2 gUnknown_80E6BF4;
-extern struct MenuItem gUnknown_80E6C0C[];
-extern struct UnkTextStruct2 gUnknown_80E6C50;
-extern struct MenuItem gUnknown_80E6C68[];
-extern struct UnkTextStruct2 gUnknown_80E6CD0;
-extern struct MenuItem gUnknown_80E6CE8[];
-extern struct UnkTextStruct2 gUnknown_80E6DDC;
-extern struct MenuItem gUnknown_80E6DF4[];
-extern struct UnkTextStruct2 gUnknown_80E6D54;
-extern struct MenuItem gUnknown_80E6D6C[];
+const struct UnkTextStruct2 gUnknown_80E653C = {
+   0x00, 0x00, 0x00, 0x00,
+   0x03,
+   0x00, 0x00, 0x00, 0x00,
+   0x00, 0x00,
+   0x00, 0x00,
+   NULL
+};
+const struct UnkTextStruct2 gUnknown_80E6554 = {
+   0x00, 0x00, 0x00, 0x00,
+   0x03,
+   0x03, 0x00, 0x08, 0x00,
+   0x18, 0x05,
+   0x05, 0x00,
+   NULL
+};
+
+const u8 sUnknown_80E65C4[];
+const u8 sUnknown_80E65AC[];
+const u8 sUnknown_80E658C[];
+const struct MenuItem gUnknown_80E656C[4] = {
+    {sUnknown_80E65C4, 0x5},
+    {sUnknown_80E65AC, 0x5},
+    {sUnknown_80E658C, 0x5},
+    {NULL, 0x3},
+
+};
+ALIGNED(4) const u8 sUnknown_80E658C[] = _("{CENTER_ALIGN}Don~27t turn the power off!");
```

Comment:
A couple more characters and formatting codes can be swapped out in this file's strings. E.g., ~27 is {APOSTROPHE} and #R is {END_COLOR_TEXT_1}.